libraries/SuplaDevice/SuplaDevice.cpp

```diff
@@ -1115,18 +1124,15 @@ void SuplaDeviceClass::iterate(void) {
   }
 
   if (!Supla::Network::Connected()) {
-    int port = 2015;
+    int port = 2015; /* deprecated in ESP8266 */
     status(STATUS_DISCONNECTED, "Not connected");
-    supla_log(LOG_DEBUG,
-              "Establishing connection with: %s (port: %d)",
-              Supla::Channel::reg_dev.ServerName,
-              port);
 
     registered = 0;
 
     if (!Supla::Network::Connect(Supla::Channel::reg_dev.ServerName, port)) {
```

Comment:
remove "port" from parameter list. Change this parameter to have default value -1

Reply:
I can’t do this because it is used by other boards like esp32, ethernet shield.
Of I do this default should be 2015 or change other network implementation. It’s not a pary of adding SSL to esp8266